pycbc/events/events.py

```diff
@@ -319,11 +319,29 @@ def keep_loudest_in_interval(self, window, num_keep):
         wtime = (time / window).astype(numpy.int32)
         bins = numpy.unique(wtime)
 
+        if log_chirp_width:
+            from pycbc.conversions import mchirp_from_mass1_mass2
+            m1 = numpy.array([p['tmplt'].mass1 for p in self.template_params])
+            m2 = numpy.array([p['tmplt'].mass2 for p in self.template_params])
+
+            # chirp mass of each template
+            mc = mchirp_from_mass1_mass2(m1, m2)[e['template_id']]
+            lmc = numpy.log(mc)
+            imc = (lmc / log_chirp_width).astype(numpy.int32)
```

Comment:
can you add a comment to this line to tell me what it does?  also perhaps can also combine with the previous line, ie `imc = (numpy.log(mc) / log_chirp_width).astype(numpy.int32)` as 'lmc' is not used in the rest of the code